src/pages/Playlists/Playlists.jsx

```diff
@@ -40,12 +41,16 @@ function Playlists() {
   const addPlaylistHandler = async () => {
     const title = newPlaylistTitle.trim();
     if (title.length === 0) {
-      alert("You must name your playlist");
+      toast.info("You must name your playlist");
       return;
     }
 
-    if (playlists.some((playlist) => playlist.title === title)) {
-      alert(`Playlist with the name '${title}' already exists!`);
+    if (
+      playlists.some(
+        (playlist) => playlist.title.toLowerCase() === title.toLowerCase()
+      )
```

Comment:
you don't need to write a function in if 
make a variable     isPlayListName  = playlists.some((playlist) => playlist.title.toLowerCase() === title.toLowerCase()

and isPlayListName is value is true if is found or else false
if(isPlayListName) {.......]
which is make code more readable